lib/internal/debugger/inspect_repl.js

```diff
@@ -1122,7 +1122,6 @@ function createRepl(inspector) {
     await Profiler.enable();
     await Profiler.setSamplingInterval({ interval: 100 });
     await Debugger.enable();
-    await Debugger.setPauseOnExceptions({ state: 'none' });
```

Comment:
Are we sure this is duplicate? Note that these are async calls. It might be needed for the next two calls (to `setAsyncCallStackDepth()` and `setBlackboxPatterns()`.

Reply:
As far as anybody on the issue thread (#41795) could tell, there shouldn't be any reason to have this duplicated call, especially since the runtime is paused until `runIfWaitingForDebugger` below and can't trigger exceptions.

There are other places in tests where we set up a `Debugger` agent and it only calls `setPauseOnException once: https://github.com/nodejs/node/blob/508890d79594bab77d4fb084d9df3284c0550b29/test/sequential/test-inspector.js#L60-L80